mod_ci/models.py

```diff
@@ -50,7 +50,6 @@ class Kvm(Base):
     test = relationship('Test', uselist=False)
     timestamp = Column(DateTime(), nullable=False)
     timestamp_prep_finished = Column(DateTime(), nullable=True)
-    timestamp_build_finished = Column(DateTime(), nullable=True)
```

Comment:
I think we would have to remove `timestamp_build_finished` from the model as well.

Reply:
I couldn't get it. Can you please explain a bit more?